cache-filter/src/filter/http.rs

```diff
@@ -96,7 +100,7 @@ impl HttpContext for CacheFilter {
                 info!("ctxt {}: Cache Miss", context_id);
                 // TODO: Avoid multiple calls for same application
                 // saving request data to use when there is response from 3scale
-                self.req_data = request_data.clone();
+                // self.req_data = request_data.clone();
```

Comment:
You need this data to be present when the response comes from 3scale.

Reply:
it's actually not removed. when you place it inside a match arm , request data binding happens only for that. so in the case of cache hit , request data is not getting binded to the HttpContext and I'm getting empty data. So I just moved the request data binding to the outer scope.